src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OwnersAndOfficersDataBlockValidator.java

```diff
@@ -87,8 +87,9 @@ private boolean isCorrectCombinationOfOwnersAndOfficersForStatement(OverseasEnti
                  }
                  break;
             case SOME_IDENTIFIED_ALL_DETAILS:
-                if (!hasBeneficialOwners(overseasEntitySubmissionDto)) {
-                    logValidationErrorMessage(errors, loggingContext, String.format("%s for statement that some can be identified", MISSING_BENEFICIAL_OWNER));
+                // TODO roe-1640 AC are that at least one BO and at least one MO must be present - if either one is missing an error is logged and returned
+                if (!(hasBeneficialOwners(overseasEntitySubmissionDto) || hasManagingOfficers(overseasEntitySubmissionDto))) {
+                    logValidationErrorMessage(errors, loggingContext, String.format("%s for statement that some can be identified", MISSING_BENEFICIAL_OWNER + " or " + MISSING_MANAGING_OFFICER));
```

Comment:
the test coverage is now a bit low for the PR (67%). looking on sonar it seems like the line that adds the error message (line 92) isn't covered by a test anymore. It is probably due to changing the unit test from a testErrorReported to a testNoErrorReported. 
I think we'll have to add a test to cover line 92 and make sure error is reported when it is supposed to.